packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -855,18 +875,24 @@ class Auth {
             },
             body
         })
-        await this.loginRegisteredUserB2C({
-            username: login,
-            password
-        })
+        await this.loginRegisteredUserB2C(
+            {
+                username: login,
+                password
+            },
+            parameters
+        )
         return res
     }
 
     /**
      * A wrapper method for commerce-sdk-isomorphic helper: loginRegisteredUserB2C.
      *
      */
-    async loginRegisteredUserB2C(credentials: LoginRegisteredUserB2CCredentials) {
+    async loginRegisteredUserB2C(
+        credentials: LoginRegisteredUserB2CCredentials,
+        parameters: helpers.CustomRequestBody = {}
```

Comment:
We should call this `body` instead of parameters to avoid confusion. Parameters seem to indicate query param string in the slas context, not the generic params. Since `body` is a field in `options` in the commerce-sdk/isomorphic, I'd suggest we use the same signature for consistency. 
```suggestion
        options?: {body: helpers.CustomRequesBody}
```

Reply:
This suggestion has been applied